src/coreclr/jit/lclvars.cpp

```diff
@@ -8052,7 +8102,7 @@ void Compiler::lvaDispVarSet(VARSET_VALARG_TP set, VARSET_VALARG_TP allVars)
                 needSpace = true;
             }
 
-            printf("V%02u", lclNum);
+            printf("[V%02u, T%d]", lclNum, index);
```

Comment:
This printing needs some fixups... Right now it is breaking alignment:
```diff
-BB02 USE(1)={    V02    } + ByrefExposed + GcHeap
-     DEF(2)={V01     V00} + ByrefExposed* + GcHeap*
+BB02 USE(1)={    [V02, T1]    } + ByrefExposed + GcHeap
+     DEF(2)={[V01, T0]     [V00, T2]} + ByrefExposed* + GcHeap*
```
Also, FWIW, I do find the local-numbers-only display easier to work with for `dCVarSet`.

Reply:
This was helpful for me to easily track the mapping during debugging. I will revert this change before merging.